web/vibe/web/rest.d

```diff
@@ -2062,10 +2062,10 @@ do {
 			}
 		}
 
-		// Check for misplaced ref / out
+		// Check for misplaced out
 		alias PSC = ParameterStorageClass;
 		foreach (i, SC; ParameterStorageClassTuple!Func) {
-			static if (SC & PSC.out_ || SC & PSC.ref_) {
+			static if (SC & PSC.out_) {
```

Comment:
following the comments on https://github.com/vibe-d/vibe.d/issues/2415#issuecomment-588079108

we could me more specific and just reject 'ref's, but allow 'in refs', but considering the long and complicated history of 'in' I chose this simple solution

checking for 'in' would require taking compiler versions and compiler flags (-preview=in) into consideration

Reply:
I suggest that instead of just allowing `ref`, only allow `ref` on a `const` parameter.
This will also cover `in [ref]` as a nice side effect, with or without the switch :)

Reply:
okay, just made a new checkin